src/Core/FlatpakBackend.vala

```diff
@@ -498,7 +498,7 @@ public class AppCenterCore.FlatpakBackend : Backend, Object {
 
         var added_remotes = new Gee.ArrayList<string> ();
 
-        uint deprecated_dependencies_number = 0;
+        bool runtime_eol = false;
```

Comment:
Okay, I see why you pulled this out into its own uint before, but I think we can revert further and just use `package.runtime_eol` directly again?

```suggestion
```